src/content/1.7/webservice/resources/addresses.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Addresses
+---
+
+# Resources for Addresses
+
+### Address
+
+|        Name         |       Format       | Required | Read Only | Max size | Not filterable | Description |
```

Comment:
Just a remark: about read only it appears most field are writable and only a handful of them are read only Besides I think people who read this documentation are more interested about things they can modify rather than those they can't So should we display a `writable` column instead of a `read-only` one

Also many rows end up with empty values, maybe it would be easier to read if we used some icons instead of `true` ? Like ✅   and ❌   (or similar) and populate each cells

I have the same questions regarding `Not filterable` column, especially when all the fields of an API appear to be filterable. Maybe we should only display this column when some fields have special values (same for read only). And add a note a the end of the table when it's not displayed like:
`All fields are filterable`
`All fields are writable`

This would make the table smaller and easier to read

Reply:
I change it with emojis as you requested, it's a good idea.
But for "All fields are filterable / writable", if we add a not writable field, we will have to change the whole doc instead of changing one line :sweat_smile: